.gitignore

```diff
@@ -1,2 +1,4 @@
 playground.el*
-.elc
\ No newline at end of file
+.elc
+\#*.*
```

Comment:
I added these, looks like my recreated  emacs configf still pollutes these :) I need to fix that..

Reply:
No problem.